Automattic/jetpack
# My Jetpack: Install Jetpack if stand-alone plugin installation  fails

#### Changes proposed in this Pull Request:

* Install Jetpack if stand-alone plugin installation  fails

#### Jetpack product discussion


p9dueE-55u-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Spin up a new JN site with Jetpack Beta
* Activate Jetpack Boost in this branch
* Make sure to deactivate and DELETE Jetpack plugin
* Go to Boost and Set up
* Go to My Jetpack
* Click Add Search
* No need to complete the authorization / purchase flow. Go back to your site and make sure the Jetpack plugin was installed
* Go back plugins and deactivate and DELETE Jetpack again

Testing Jetpack Search plugin activation. This needs to be done in the monorepo local environment
* Make sure both Jetpack and Search are not activated
* Go to My Jetpack
* Add Search
* Make sure Search plugin was activated - and not Jetpack
